src/github/client/github-queries.ts

```diff
@@ -78,7 +78,50 @@ export const getPullRequests = `query ($owner: String!, $repo: String!, $per_pag
     }
   }`;
 
-export const getCommits = (includeChangedFiles?: boolean) => `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor: String, $default_ref: String!) {
+export type getCommitsResponse = {
```

Comment:
Didn't have an appropriate Octokit type that I could find, felt like a unique case so made a custom Type.

Reply:
Yeah, most graphql ones are going to be unique, sadly.